server/api-util/lineItems.js

```diff
@@ -2,7 +2,9 @@ const { calculateQuantityFromDates, calculateTotalFromLineItems } = require('./l
 const { types } = require('sharetribe-flex-sdk');
 const { Money } = types;
 
-const unitType = 'line-item/night';
+// This bookingUnitType needs to be one of the following:
+// line-item/night, line-item/day or line-item/units
+const bookingUnitType = 'line-item/night';
```

Comment:
I changed the name so that it's in sync with the config.js variable (in web app).